docs/upctl_server_restart.md

```diff
@@ -0,0 +1,30 @@
+## upctl server restart
+
+Restart a server
+
+```
+upctl server restart <UUID/Title/Hostname...> [flags]
+```
+
+### Options
+
+```
+      --stop-type string   Restart type. Available: soft, hard (default "soft")
```

Comment:
should this be --restart-type? i guess stop isnt too bad either but the documentation could then refer to 'stop type', maybe?